assets/js/pages/Users/UsersPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { listUsers, deleteUser } from '@lib/api/users';
+
+import { toast } from 'react-hot-toast';
+
+import Users from './Users';
+
+const successDeleteMessage = 'User deleted successfully';
+const errorLoadingMessage = 'An error occurred during loading users';
+const errorDeletingMessage = 'An error occurred during deleting user';
+
+function UsersPage() {
+  const [loading, setLoading] = useState(false);
```

Comment:
Not a big deal, but maybe we should put loading to `true` at the beginning, so we don't have any possibility that we pass through any strange state

Reply:
Actually, it is a very good idea…
The API call is fast, loading state does not look that good but if loading is per default true, it makes the transition better for giving it a little bit more time !